src/mofuw.nim

```diff
@@ -55,6 +58,24 @@ var
   callback    {.threadvar.}: Callback
   bufferSize  {.threadvar.}: int
 
+proc countCPUs(): int =
+  when defined(linux):
```

Comment:
Can I use `... and defined(glibc)` ?

Reply:
can use `when defined(glibc)`

Reply:
sorry.
`when defined ()` seems to be compiled regardless of what we pass
There is a possibility that it is not the correct operation assumed.

Reply:
Thanks. Hmm, it seems there's no way to determine what libc using...

Reply:
Well, i think probably will not be using Nim with older glibc and musl-libc...
merge this PR's. Thanks :)